Mathlib/FieldTheory/IsAlgClosed/Basic.lean

```diff
@@ -209,8 +209,16 @@ instance (priority := 100) IsAlgClosure.separable (R K : Type*) [Field R] [Field
 
 namespace IsAlgClosed
 
-variable (K : Type u) [Field K] (L : Type v) (M : Type w) [Field L] [Algebra K L] [Field M]
-  [Algebra K M] [IsAlgClosed M] (hL : Algebra.IsAlgebraic K L)
+variable {K : Type u} [Field K] {L : Type v} {M : Type w} [Field L] [Algebra K L] [Field M]
+  [Algebra K M] [IsAlgClosed M]
+
+theorem surjective_comp_algebraMap_of_isAlgebraic {E : Type*}
```

Comment:
Can you add a docstring to this one (and to the other version)? Maybe also a reference, for example is this Lemma 4.13 of http://alpha.math.uga.edu/~pete/FieldTheory.pdf ?